### PR TITLE
Toggle between plain text and HTML mode.

### PR DESCRIPTION
**Pull request by spchand86**
Included setting for switching to plain text mode.
Related issue #1397 .

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageContainerView.java [hunk 1]
```diff
@@ -388,7 +389,10 @@ public void displayMessageViewContainer(MessageViewInfo messageViewInfo,
             mSavedState = null;
         }
 
-        String textToDisplay = messageViewInfo.text;
+        String textToDisplay = K9.displayAsPlaintText()
+                ? HtmlConverter.htmlToText(messageViewInfo.text)
```
Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageContainerView.java]: This is not what we want to show. The text displayed here should be the `text` field from `ViewableExtractedText`, please see `MessageViewInfoExtractor` for the relevant code.

**Comment by PokerFace8** [comment]
@Valodim , I used the text field from ViewableExtractedText, but for doing so I had to change the MessageViewInfo class to include a String variable for the plainText. Is it the right way to go about things?

**Comment by dhebbeker** [comment]
This pull request may partially resolve #906.

**Comment by cketti** [comment]
Thanks for attempting to implement this feature. But as I mentioned [here](https://github.com/k9mail/k-9/issues/906#issuecomment-462871300) I don't want to maintain a separate code path that uses a `TextView`.

**Pull request closed without merging**